### DISCUSSION TITLE
May i join the Team?

### DISCUSSION THREAD

**Discussion by Amritokarmokar** [ROM-builders/temporary#23736] [General]
Hi sir good day 😊 

Here Is Some Of My Successful Builds of my main account which suddenly got banned

PixelOS
https://cirrus-ci.com/build/4991486597529600

Project-404
https://cirrus-ci.com/build/6641334732193792

Here Is My Telegram account
https://t.me/Amritorock

My tree's
https://github.com/Amritorock/device_realme_r5x
https://github.com/Amritorock/vendor_realme_r5x
https://github.com/Amritorock/kernel_realme_r5x

My GitHub account 
https://github.com/Amritokarmokar

**Comment by Apon77** [comment]
You should not misuse cirrus. 
need to wait some days.

**Reply by Amritokarmokar** [reply]
Okay sir I'll wait until you give me access again thanks.
